src/styles/blocks/page.scss

```
@@ -0,0 +1,35 @@
+* {
+  margin: 0;
+  color: $main-accent-color;
+  transition: transform 0.3s;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.content {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  row-gap: 46px;
+  column-gap: 48px;
+  justify-content: center;
+  padding-top: 50px;
+  padding-bottom: 72px;
+
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, 200px);
+    }
+  }
```

Redundant, need to fix it everywhere
```suggestion
  @media (min-width: 488px) {
    grid-template-columns: repeat(2, 200px);
  }
```